CHANGES.rst

```diff
@@ -56,6 +56,9 @@ Breaking changes in 0.3.0
   renamed to `USBStorageDriver`.
   A deprecated `NetworkUSBStorageDriver` exists temporarily for compatibility
   reasons.
+- ``codec`` and ``decodeerrors`` are no longer ``run()``/``run_check()``
+  kwargs, but driver (``codec``) and `CommandMixin` (``decodeerrors``)
+  attributes.
```

Comment:
Why is codec a driver attribute and decodeerrors on CommandMixin?

This should also describe how a user should update their code and configuration. Is the encoding changeable at runtime?